### DISCUSSION TITLE
Add calculate the cost of kwh withdrawn in main Stats

### DISCUSSION THREAD

**Discussion by SiNaPsEr0x** [teslamate-org/teslamate#1502] [Ideas]
could the calculation of the kwh actually taken from the network be included? in this way it is known more precisely how much one has absorbed from the meter since this data is already present in the individual refills.

thanks for your jobs and I donate it ;)

**Comment by adriankumpf** [comment]
Moving this to [💡 Idea](https://github.com/adriankumpf/teslamate/discussions/categories/ideas). The  kWh actually taken from the network is not available through the Tesla API though.
